src/content/whats-new/2022/03/whats-new-02-28-EKS-Add-on.md

```diff
@@ -0,0 +1,19 @@
+---
+title: 'Automate Kubernetes observability on your EKS clusters with code'
+summary: 'Deploy observability faster and easier than ever!'
+releaseDate: '2022-02-28'
+learnMoreLink: 'https://newrelic.com/blog/nerdlog/eks-ssp-kubernetes-observability' 
+getStartedLink: 'https://www.npmjs.com/package/@newrelic/newrelic-ssp-addon'
+
```

Comment:
Remove space

Reply:
Need need to add code for new K8s_apm image

Reply:
@daynaslord , I removed that blank line in the front matter. Also, I pushed up the new image.